CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 - Major: Potentially breaking, filters in commands are now applied even if the result of a substitution returns an error. (#1973)  
   This makes all `or_...` filters a lot more useful, but it may mean that some other filters will need some additional error handling.  
   This will require some experimentation, reporting errors in our GitHub issues for this is greatly appreciated.
+- Minor: Added `name_and_login` user property, which displays a user's login name if their display name is not in English. (#1873)
```

Comment:
```suggestion
- Minor: The website now shows the user's login name in addition to the display name if the user's name is not in Latin characters. (#1873)
```

Reply:
Wouldn't this make it seem like it's replaced in EVERY scenario, when it's actually not?

Reply:
It does it in as many places as makes sense for now - we can fill in the gap if we find a place where it makes sense to do it after this PR